chcemvediet/apps/obligees/tests/test_forms.py

```diff
@@ -115,102 +121,106 @@ def test_submitted_with_nonexisting_obligee_name(self):
                 <input class="autocomplete" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text" value="invalid">
                 """.format(url=reverse(u'obligees:autocomplete')), rendered)
 
-    @unittest.skip(u'FIXME')
-    def test_to_python_is_cached(self):
-        names = [u'aaa', u'bbb', u'ccc', u'ddd']
-        oblgs = [self._create_obligee(name=n) for n in names]
-        field = ObligeeField()
-
-        # Valid value
-        with self.assertNumQueries(1):
-            self.assertEqual(field.clean(u'bbb'), oblgs[1])
-        with self.assertNumQueries(0):
-            self.assertEqual(field.clean(u'bbb'), oblgs[1])
-
-        # Invalid value
-        with self.assertNumQueries(1):
-            with self.assertRaises(ValidationError):
-                field.clean(u'invalid')
-        with self.assertNumQueries(0):
-            with self.assertRaises(ValidationError):
-                field.clean(u'invalid')
-
 class ObligeeFieldWithObligeeWidgetWidget(ObligeesTestCaseMixin, TestCase):
     u"""
     Tests ``ObligeeField`` with ``ObligeeWidget`` widget.
     """
 
     class Form(forms.Form):
-        obligee = ObligeeField(
-                widget=ObligeeWidget(),
-                )
+        obligee = ObligeeField()
 
     class FormWithWidgetAttrs(forms.Form):
         obligee = ObligeeField(
-                widget=ObligeeWidget(attrs={
+                widget=ObligeeWidget(input_attrs={
                     u'class': u'custom-class',
                     u'custom-attribute': u'value',
                     }),
                 )
 
 
-    @unittest.skip(u'FIXME')
```

Comment:
V subore `obligees/forms.py`, vo funkcii `coerce` je comment ```"# FIXME: Should be .get(name=value), but there are Obligees with duplicate names, yet."``` Je to stale aktualne, mozu existovat Obligee instancie s rovnakym nazvom? field `name` je definovany ako `models.CharField(max_length=255, unique=True, ...)`

Reply:
@viliambalaz Pravdu mas, uz to nie je aktualne. Treba `filter(name=value).order_by_pk().first()` prepisat na `.get(name=value)`. A upravit handlovanie pripadu, ked taky obligee v DB nie je. Kedze `.get()` raisne exception v tom pripade. Pls fixni to, ale v samostatnom PR.

Reply:
:ok: